pkg/resources/custom_diffs.go

```diff
@@ -67,6 +67,29 @@ func BoolComputedIf(key string, getDefault func(client *sdk.Client, id sdk.Accou
 	})
 }
 
+// SetEmptyForceNewIfChange sets a ForceNew for a list field which was set to an empty value.
+func SetEmptyForceNewIfChange[T any](key string) schema.CustomizeDiffFunc {
+	return customdiff.ForceNewIfChange(key, func(ctx context.Context, oldValue, newValue, meta any) bool {
+		oldList, newList := oldValue.([]T), newValue.([]T)
+		return len(oldList) > 0 && len(newList) == 0
+	})
+}
+
+// SetEmptyForceNewIfChangeString sets a ForceNew for a string field which was set to an empty value.
+func SetEmptyForceNewIfChangeString(key string) schema.CustomizeDiffFunc {
+	return customdiff.ForceNewIfChange(key, func(ctx context.Context, oldValue, newValue, meta any) bool {
+		oldString, newString := oldValue.(string), newValue.(string)
+		return len(oldString) > 0 && len(newString) == 0
+	})
+}
+
+// SetForceNewIfNull sets a ForceNew for key missing in RawConfig.
+func SetForceNewIfNull(key string) schema.CustomizeDiffFunc {
+	return customdiff.ForceNewIf(key, func(ctx context.Context, d *schema.ResourceDiff, meta interface{}) bool {
+		return d.GetRawConfig().AsValueMap()[key].IsNull()
```

Comment:
hmmmm, I don't think this is good. It will trigger every time, right (even with no change: empty -> empty).

Reply:
This is not needed - removed.